lib/src/messaging_client.dart

```diff
@@ -16,8 +18,6 @@ abstract class MessagingClient {
   /// If the application does not support messaging, this will throw an exception.
   Stream<Message> get messageStream;
 
-  Future<void> sendStart();
```

Comment:
Removed `sendStart` from the public API, as it's a legacy solution that we want to eliminate.